steps/27/webapp/test/Test.cdn.qunit.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<script
+		src="https://sdk.openui5.org/1.129.0/resources/sap/ui/test/starter/runTest.js"
```

Comment:
```suggestion
		src="https://sdk.openui5.org/1.130.0/resources/sap/ui/test/starter/runTest.js"
```
Should we use the latest published UI5 version?

Reply:
The whole tutorial currently uses 1.129, so I didn't want to change that in this PR. An update to 1.130 makes sense to me but I'd rather do it in a separate PR.

Reply:
Thx a lot for clarification.